style1.css

```diff
@@ -0,0 +1,235 @@
+/* Tablet+ */
+body {
+background: linear-gradient(90deg, #83EB94 0%, #47D95E 100%);
+min-width: 768px;
+margin: 4px;
+display: block;
+}
+
+.container {
+width: 768px;
+height: 1024px;
+margin-left: auto;
+margin-right: auto;
+}
+
+/* coming-soon-ribbon */
+.coming-soon {
+position: absolute;
+left: 83.85%;
+right: 0%;
+top: 0.1%;
+bottom: 81.45%;
+
+background: url(coming-soon-ribbon.png) no-repeat; 
+}
+
+/* Header */
+
+header {
+overflow: visible;
+}
+
+.header_image {
+margin: 64 40.8 652 40.8;
+
+}
+
+h1 {
+margin: 0%;
+height: 100px;
+width: 768px;
+
+font-family: 'Lato', sans-serif;
+font-style: normal;
+font-weight: 300;
+line-height: normal;
+font-size: 60px;
+text-align: center;
+letter-spacing: 5px;
+
+color: #FFFFFF;
+
+mix-blend-mode: normal;
+}
+
+/* Form */
+
+form {
+left: 0.52%;
+right: -0.52%;
+top: 45.8%;
+bottom: 34.95%;
+}
+
+/* Form Title */
+
+p {
+left: 0%;
+right: 0%;
+top: 0%;
+bottom: 75.03%;
+
+font-family: 'Lato', sans-serif;
+font-style: normal;
+font-weight: normal;
+line-height: normal;
+font-size: 24px;
+text-align: center;
+
```

Comment:
Musisz powtarzać kod z tym wyborem czcionki? To nie są 2 linijki że można przymknąć oko... 
Lepiej dać to choćby w body... Lub jakiś element, który jest rodzicem obu tych elementów.